cli/azd/cmd/root.go

```diff
@@ -175,7 +175,13 @@ func runCmdWithTelemetry(cmd *cobra.Command, runCmd func(ctx context.Context) er
 	// Note: CommandPath is constructed using the Use member on each command up to the root.
 	// It does not contain user input, and is safe for telemetry emission.
 	spanCtx, span := telemetry.GetTracer().Start(cmd.Context(), events.GetCommandEventName(cmd.CommandPath()))
-	defer span.End()
+	end := func() {
+		// Include any usage attributes set
+		span.SetAttributes(telemetry.GetUsageAttributes()...)
```

Comment:
Should we move this logic into `wrapperSpan`'s `End` method as we do for global attributes?

Reply:
The way I envision `usage` attributes to work is that it is consumed by usage-type events, hence why it's added here. For example, it is noisy to include `templateId` on all events. It simply isn't that important.